components/editor/modules/chart/EditOverlay/config.js

```diff
@@ -380,7 +380,7 @@ year,category,value
       type: 'Line',
       unit: 'Jahre',
       numberFormat: '.1f',
-      zero: false,
+      zero: true,
```

Comment:
if you want a line chart starting at zero we should add a different example data set that does not just move between 75 and 83 :)

Reply:
you are right, I will add another example